### DISCUSSION TITLE
specify trace path when launching nextflow

### DISCUSSION THREAD

**Discussion by fgualdr** [nextflow-io/nextflow#4290] [Q&A]
Hi,
I have a general question.
Is it possible to specify the trace.txt path when executing a workflow?
I am executing nextflow run with the "-with-trace" option but I would like to change the writing location.
Is it possible to do this at the run stage? like "-with-trace ../path/trace.txt"?
Thanks for the help
Francesco

**Comment by mribeirodantas** [comment]
Yes, check [here](https://www.nextflow.io/docs/latest/cli.html?highlight=trace).

You can totally do something like:

```Groovy
nextflow run nextflow-io/hello -with-trace test/teste.txt
```

But be aware that by default, Nextflow will not overwrite already existing trace files, so if you want it to overwrite previously created trace files, you must set `trace.overwrite` to true in a configuration file such as `nextflow.config`.
